Mathlib/Algebra/Algebra/Equiv.lean

```diff
@@ -620,6 +618,8 @@ end OfRingEquiv
 
 -- Porting note: projections mul & one not found, removed [simps] and added theorems manually
 -- @[simps (config := .lemmasOnly) one]
+@[stacks 09HR "This generalize beyond the field case in 09HR. In mathlib, we use `AlgEquiv F E E` \
+to denote the automorphism group Aut(E/F)."]
```

Comment:
I think this comments needs to be amended (or removed)

Reply:
Thanks! I'll remove the comments.